### ISSUE TITLE
Video and audio size and duration are not announced

### ISSUE THREAD

**Issue by FrayxRulez** [UnigramDev/Unigram#2322]
Currently size and duration for video and audio messages are not announced

**Comment by sukiletxe** [comment]
The size fix posted today doesn't work in the media tab (unless 6136 doesn't have that implemented).
